script.js

```diff
@@ -1,4 +1,339 @@
+        "/" +
+        gameCount +
+        " <br>You have " +
+        currentDraw +
+        " Draws";
+      return drawMessage + "<br>" + winLossCountMessage;
+    }
+    //scissors win rock
+    if (input == "scissors" && assignNumberToOption == "rock") {
+      currentWin = currentWin + 1;
+      gameCount = gameCount + 1;
+      winLossCountMessage =
+        "Your Win/Loss is " +
+        currentWin +
+        "/" +
+        gameCount +
+        " <br>You have " +
+        currentDraw +
+        " Draws";
+      return winMessage + "<br>" + winLossCountMessage;
+    }
+    //paper lose scissors
+    if (input == "paper" && assignNumberToOption == "scissors") {
+      currentWin = currentWin + 1;
+      gameCount = gameCount + 1;
+      winLossCountMessage =
+        "Your Win/Loss is " +
+        currentWin +
+        "/" +
+        gameCount +
+        " <br>You have " +
+        currentDraw +
+        " Draws";
+      return winMessage + "<br>" + winLossCountMessage;
+    }
+    //rock win paper
+    if (input == "rock" && assignNumberToOption == "paper") {
+      currentWin = currentWin + 1;
+      gameCount = gameCount + 1;
+      winLossCountMessage =
+        "Your Win/Loss is " +
+        currentWin +
+        "/" +
+        gameCount +
+        " <br>You have " +
+        currentDraw +
+        " Draws";
+      return winMessage + "<br>" + winLossCountMessage;
+    }
+    if (input != "") {
+      return invalidMessage;
+    }
+    return emptyMessage;
+  } else if (currentGameMode == "SPR") {
+    //Option is Computer Generated Result of RPS
+    var assignNumberToOption = randomiser();
+    console.log(assignNumberToOption);
+    //Messages
+    var winMessage = "You Win,";
+    var drawMessage = "Draw!";
+    var loseMessage = "You Lose!";
+    var emptyMessage = "Please enter scissors/paper/rock into box";
+    var invalidMessage = "Invalid entry!!!!";
+    var winLossCountMessage =
+      ", Your Win/Loss is " +
+      currentWin +
+      "/" +
+      gameCount +
+      " You have " +
+      currentDraw +
+      " Draws";
+    //scissors win paper
+    if (input == "scissors" && assignNumberToOption == "paper") {
+      currentWin = currentWin + 1;
+      gameCount = gameCount + 1;
+      winLossCountMessage =
+        "Your Win/Loss is " +
+        currentWin +
+        "/" +
+        gameCount +
+        " <br>You have " +
+        currentDraw +
+        " Draws";
+      return winMessage + "<br>" + winLossCountMessage;
+    }
+    //paper win rock
+    if (input == "paper" && assignNumberToOption == "rock") {
+      currentWin = currentWin + 1;
+      gameCount = gameCount + 1;
+      winLossCountMessage =
+        "Your Win/Loss is " +
+        currentWin +
+        "/" +
+        gameCount +
+        " <br>You have " +
+        currentDraw +
+        " Draws";
+      return winMessage + "<br>" + winLossCountMessage;
+    }
+    //rock win scissors
+    if (input == "rock" && assignNumberToOption == "scissors") {
```

Comment:
again ,combine this condition with the other condition using Boolean operator ||